common/HIJetReco.C

```diff
@@ -29,6 +32,9 @@ namespace Enable
   int HIJETS_VERBOSITY = 0;
   bool HIJETS_MC = true;
   bool HIJETS_TRUTH = false;
+  bool HIJETS_TOWER = true;   // make tower jets
+  bool HIJETS_TRACK = false;  // make track jets
+  bool HIJETS_PFLOW = false;  // make particle flow jets
 }  // namespace Enable
```

Comment:
TO-DO: add doBkgdSub flag?

Reply:
Since there use-cases for running with jet reconstruction without background subtraction, I opted to instead add a parallel "no subtraction" macro, `NoBkgdSubJetReco.C`. My hope is that this will make it more clear what workflow each of these macros is for.

For `HIJetReco`, though, I've added warnings that track/PF background subtraction is still under development if the user tries to run track/PF jet reconstruction in HIJetReco.